src/ripple/protocol/impl/ErrorCodes.cpp

```diff
@@ -26,105 +27,96 @@ namespace RPC {
+    {rpcALREADY_SINGLE_SIG,     "alreadySingleSig",     "Already single-signed."},
+    {rpcAMENDMENT_BLOCKED,      "amendmentBlocked",     "Amendment blocked, need upgrade.", 503},
+    {rpcEXPIRED_VALIDATOR_LIST, "unlBlocked",           "Validator list expired.", 503},
+    {rpcATX_DEPRECATED,         "deprecated",           "Use the new API or specify a ledger range."},
+    {rpcBAD_KEY_TYPE,           "badKeyType",           "Bad key type."},
+    {rpcBAD_FEATURE,            "badFeature",           "Feature unknown or invalid.", 500},
+    {rpcBAD_ISSUER,             "badIssuer",            "Issuer account malformed."},
+    {rpcBAD_MARKET,             "badMarket",            "No such market."},
+    {rpcBAD_SECRET,             "badSecret",            "Secret does not match account."},
+    {rpcBAD_SEED,               "badSeed",              "Disallowed seed."},
+    {rpcBAD_SYNTAX,             "badSyntax",            "Syntax error.", 400},
+    {rpcCHANNEL_MALFORMED,      "channelMalformed",     "Payment channel is malformed."},
+    {rpcCHANNEL_AMT_MALFORMED,  "channelAmtMalformed",  "Payment channel amount is malformed."},
+    {rpcCOMMAND_MISSING,        "commandMissing",       "Missing command entry."},
+    {rpcDB_DESERIALIZATION,     "dbDeserialization",    "Database deserialization error."},
+    {rpcDST_ACT_MALFORMED,      "dstActMalformed",      "Destination account is malformed."},
+    {rpcDST_ACT_MISSING,        "dstActMissing",        "Destination account not provided."},
+    {rpcDST_ACT_NOT_FOUND,      "dstActNotFound",       "Destination account not found."},
+    {rpcDST_AMT_MALFORMED,      "dstAmtMalformed",      "Destination amount/currency/issuer is malformed."},
+    {rpcDST_AMT_MISSING,        "dstAmtMissing",        "Destination amount/currency/issuer is missing."},
+    {rpcDST_ISR_MALFORMED,      "dstIsrMalformed",      "Destination issuer is malformed."},
+    {rpcEXCESSIVE_LGR_RANGE,    "excessiveLgrRange",    "Ledger range exceeds 1000."},
+    {rpcFORBIDDEN,              "forbidden",            "Bad credentials.", 403},
+    {rpcFAILED_TO_FORWARD,      "failedToForward",      "Failed to forward request to p2p node", 503},
+    {rpcHIGH_FEE,               "highFee",              "Current transaction fee exceeds your limit."},
+    {rpcINTERNAL,               "internal",             "Internal error.", 500},
+    {rpcINVALID_LGR_RANGE,      "invalidLgrRange",      "Ledger range is invalid."},
+    {rpcINVALID_PARAMS,         "invalidParams",        "Invalid parameters.", 400},
+    {rpcJSON_RPC,               "json_rpc",             "JSON-RPC transport error.", 500},
+    {rpcLGR_IDXS_INVALID,       "lgrIdxsInvalid",       "Ledger indexes invalid."},
+    {rpcLGR_IDX_MALFORMED,      "lgrIdxMalformed",      "Ledger index malformed."},
+    {rpcLGR_NOT_FOUND,          "lgrNotFound",          "Ledger not found."},
+    {rpcLGR_NOT_VALIDATED,      "lgrNotValidated",      "Ledger not validated.", 202},
```

Comment:
Are we sure 202 is correct here? That's a "successful" class code so I wouldn't expect an error